content/en/docs/instrumentation/go/exporters.md

```diff
@@ -2,56 +2,178 @@
 title: Exporters
 aliases: [/docs/instrumentation/go/exporting_data]
 weight: 50
-cSpell:ignore: otlptrace otlptracehttp
+# prettier-ignore
+cSpell:ignore: otlpmetric otlpmetricgrpc otlpmetrichttp otlptrace otlptracegrpc otlptracehttp promhttp stdouttrace
 ---
 
 {{% docs/instrumentation/exporters-intro go %}}
 
-## OTLP endpoint
+## Console
+
+### Console traces
+
+[`go.opentelemetry.io/otel/exporters/stdout/stdouttrace`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/stdout/stdouttrace)
+contains an implementation of the console trace exporter.
+
+Here's how you can create an exporter with default configuration:
+
+```go
+import (
+	"go.opentelemetry.io/otel/exporters/stdout/stdouttrace"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+func newExporter() (trace.SpanExporter, error) {
+	return stdouttrace.New()
+}
+```
+
+### Console metrics (Experimental)
+
+[`go.opentelemetry.io/otel/exporters/stdout/stdoutmetric`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/stdout/stdoutmetric)
+contains an implementation of the console metrics exporter.
+
+Here's how you can create an exporter with default configuration:
+
+```go
+import (
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutmetric"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+func newExporter() (metric.Exporter, error) {
+	return stdoutmetric.New()
+}
+```
+
+## OTLP
 
 To send trace data to an OTLP endpoint (like the [collector](/docs/collector) or
 Jaeger >= v1.35.0) you'll want to configure an OTLP exporter that sends to your
 endpoint.
 
-### Using HTTP
+To learn more on how to use the OTLP HTTP exporter, try the
+[otel-collector example](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector)
+
+### OTLP traces over HTTP
+
+[`go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp)
+contains an implementation of the OTLP trace exporter using HTTP with binary
+protobuf payloads.
+
+Here's how you can create an exporter with default configuration:
 
 ```go
 import (
-  	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
-  	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+	"go.opentelemetry.io/otel/sdk/trace"
 )
 
-func installExportPipeline(ctx context.Context) (func(context.Context) error, error) {
+func newExporter(ctx context.Context) (trace.SpanExporter, error) {
 	client := otlptracehttp.NewClient()
-	exporter, err := otlptrace.New(ctx, client)
-	if err != nil {
-		return nil, fmt.Errorf("creating OTLP trace exporter: %w", err)
-	}
-  	/* … */
+	return otlptrace.New(ctx, client)
```

Comment:
Can we just use [`otlptracehttp.New`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp#New) here instead?

Reply:
Same for grpc?

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/3320/commits/98ded261e59b5f873809a3ffaef38fad2762abee